cmd/resim/commands/batch.go

```diff
@@ -26,33 +26,31 @@ var (
 		Long:  ``,
 		Run:   createBatch,
 	}
-)
-
-// This function takes a comma-separated list of UUIDs represented as strings
-// and returns a separated array of parsed UUIDs.
-func parseUUIDs(commaSeparatedUUIDs string) []openapi_types.UUID {
-	if commaSeparatedUUIDs == "" {
-		return []openapi_types.UUID{}
+	getBatchCmd = &cobra.Command{
+		Use:   "get",
+		Short: "get - Retrieves a batch",
+		Long:  ``,
+		Run:   getBatch,
 	}
-	strs := strings.Split(commaSeparatedUUIDs, ",")
-	result := make([]openapi_types.UUID, len(strs))
 
-	for i := 0; i < len(strs); i++ {
-		id, err := uuid.Parse(strings.TrimSpace(strs[i]))
-		if err != nil {
-			log.Fatal(err)
-		}
-		result[i] = id
-	}
-	return result
-}
+	batchIDString string
+	batchName     string
+	exitStatus    bool
+)
 
 func init() {
 	createBatchCmd.Flags().String("build_id", "", "The ID of the build.")
 	createBatchCmd.Flags().String("experience_ids", "", "Comma-separated list of experience ids to run.")
 	createBatchCmd.Flags().String("experience_tag_ids", "", "Comma-separated list of experience tag ids to run.")
 	viper.BindPFlags(createBatchCmd.Flags())
 	batchCmd.AddCommand(createBatchCmd)
+
+	getBatchCmd.Flags().StringVar(&batchIDString, "batch_id", "", "The ID of the batch to retrieve.")
+	getBatchCmd.Flags().StringVar(&batchName, "batch_name", "", "The name of the batch to retrieve.")
```

Comment:
neat!

(nit) might be worth providing an example friendly name so users know what pattern to expect? rejoicing-aquamarine-starfish

Reply:
Done.